services/server/config/config_test.go

```diff
@@ -207,3 +207,29 @@ version = 2
 	assert.NilError(t, err)
 	assert.Equal(t, true, pluginConfig["shim_debug"])
 }
+
+// TestDecodePluginInV1Config tests decoding non-versioned
```

Comment:
this test will also pass even if platformAgnosticDefaultConfig() returns Version to 2

But it is a good test to ensure v1 will pass v2 validate by it returning nil..

hmm.. let's also add a comment where the default config Version is defined explaining why it should not be changed... I'll suggest a comment above